src/scss/social-feeds/_scrollbox.scss

```diff
@@ -1,3 +1,22 @@
+$color-facebook: #3a5795;
```

Comment:
These variables can be removed (they're already defined in src/scss/_variables.scss)